src/main/blackbox/blackbox.c

```diff
@@ -1657,7 +1657,8 @@ void handleBlackbox(timeUs_t currentTimeUs)
 
 static bool canUseBlackboxWithCurrentConfiguration(void)
 {
-    return feature(FEATURE_BLACKBOX);
+    return feature(FEATURE_BLACKBOX) && 
+        (blackboxConfig()->device != BLACKBOX_SDCARD || feature(FEATURE_SDCARD));
```

Comment:
Makes sense. I'll update. Can you add in the configurator that if the feature sdcard is not turned on, and the person selects sdcard a prompt to enable it perhaps? Thoughts?